maxim/tests/test_openai.py

```
@@ -50,17 +50,17 @@ async def test_async_chat_completions_stream(self):
-        
+
         # Collect all chunks
         chunks = []
         async for chunk in response:
             chunks.append(chunk)
             if chunk.choices and len(chunk.choices) > 0 and chunk.choices[0].delta.content:
                 print(chunk.choices[0].delta.content, end="", flush=True)
-        
+
         # Verify we got some content
         self.assertTrue(len(chunks) > 0, "No chunks received from the stream")
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)